wa/workloads/jetnews/uiauto/app/src/main/java/com/arm/wa/uiauto/jetnews/UiAutomation.java

```diff
@@ -0,0 +1,81 @@
+/*    Copyright 2014-2024 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.arm.wa.uiauto.jetnews;
+
+import androidx.test.uiautomator.UiObject;
+import androidx.test.uiautomator.UiSelector;
+
+import android.os.Bundle;
+
+import com.arm.wa.uiauto.ApplaunchInterface;
+import com.arm.wa.uiauto.BaseUiAutomation;
+import com.arm.wa.uiauto.UiAutoUtils;
+
+import org.junit.Test;
+
+// Dummy workload for jetnews. We need to use JankTestBasem but we
```

Comment:
nit: JankTestBasem -> JankTestBase?

Reply:
Oops. I'll go through these comments again to see if I missed anything else.